pkg/command/build.go

```diff
@@ -86,6 +90,11 @@ func (opts *dockerfileOptions) Validate(cmd *cobra.Command, args []string) error
 		return apis.ErrMissingField("dockerfile")
 	}
 
+	if opts.Dockerfile == "" {
+		return apis.ErrMissingField("dockerfile")
+	}
```

Comment:
This looks like bad copy/pasta?

Reply:
doh, great catch - no idea where that even came from :)